eclair-core/src/main/scala/fr/acinq/eclair/router/Graph.scala

```diff
@@ -306,7 +306,7 @@ object Graph {
       val factor = (cltvFactor * wr.cltvDeltaFactor) + (ageFactor * wr.ageFactor) + (capFactor * wr.capacityFactor)
       val edgeWeight = if (isNeighborTarget) prev.weight else edgeCost * factor
 
-      RichWeight(cost = edgeCost, length = prev.length + 1, cltv = prev.cltv + channelCltvDelta, weight = edgeWeight)
+      RichWeight(cost = edgeCost, length = prev.length + 1, cltv = prev.cltv + channelCltvDelta, weight = prev.weight + edgeWeight)
```

Comment:
It would be more consistent to use instead:
```scala
      val edgeWeight = if (isNeighborTarget) prev.weight else prev.weight + edgeCost * factor

      RichWeight(cost = edgeCost, length = prev.length + 1, cltv = prev.cltv + channelCltvDelta, weight = edgeWeight)
```

Reply:
Addressed in 5039da3